relion/convert/__init__.py

```diff
@@ -144,27 +145,12 @@ def _updateParticle(self, item, row):
         item.setClassId(row.rlnClassNumber)
         self._reader.setParticleTransform(item, row)
 
-        # Try to create extra objects only once if item is reused
```

Comment:
groupname/number is now in extraLabels which is always parsed if exists for the first part / row. I do not see the use of extra ML-related labels that decrease overall conversion performance and also create unnecessary Scipion objects (Float)